openstack/path_static_role_test.go

```diff
@@ -292,7 +291,7 @@ func TestStaticRoleDelete(t *testing.T) {
 	})
 
 	t.Run("error-get", func(t *testing.T) {
-		t.Parallel()
+		//t.Parallel()
```

Comment:
Don't comment out code, just delete it. If there is some problem restricting parallel usage, it's better to add a comment for that, describing the problem